### PR TITLE
(rack-cors): Fix problem access from mobile

### PR DESCRIPTION
**Pull request by yohanology**

### CODE REVIEW

**Comment by h8rry** [comment]
Read the documentations carefully: https://github.com/cyu/rack-cors

The Rack section is separated from the Rails section.

**Comment by yohanology** [comment]
It's working... what do I need to change??

File: config.ru [hunk 1]
```diff
@@ -1,4 +1,25 @@
 # This file is used by Rack-based servers to start the application.
```
Comment (ggharry) [inline config.ru]: This file is not needed

File: config.ru [hunk 2]
```diff
@@ -1,4 +1,25 @@
...
 
+use Rack::Cors do
```
Comment (h8rry) [inline config.ru]: These changes are not needed for Rails. It's only for Rack.

File: config/application.rb [hunk 3]
```diff
@@ -30,6 +30,13 @@ class Application < Rails::Application
     # config.i18n.default_locale = :de
 
     # Do not swallow errors in after_commit/after_rollback callbacks.
+    config.middleware.insert_before 0, "Rack::Cors" do
```
Comment (h8rry) [inline config/application.rb:33]: This is good to keep. It's for Rails.

**Comment by h8rry** [comment]
You can merge once it's done.

**Pull request merged**